marketplace/core/models.py

```diff
@@ -43,5 +48,12 @@ class AppTypeBaseModel(BaseModel):
 
     code = models.SlugField(validators=[validate_app_code_exists])
 
+    @property
+    def apptype(self) -> "AppType":
+        """
+        Returns de respective AppType
```

Comment:
pls, change the string `Returns the respective AppType`

Reply:
Thanks for the review 😄